reactDashboard/src/components/DashboardNavbar/DashboardNavbar.css

```diff
@@ -68,19 +137,25 @@
 }
 
 @media screen and (max-width: 575px) {
```

Comment:
Having some trouble fixing the font awesome icon at 575px. I haven't figured out how to make one icon disappear at that screen size.

Reply:
nvm, fixed it